app/src/main/res/layout/activity_main.xml

```diff
@@ -1,18 +1,22 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+    android:orientation="vertical"
     tools:context=".MainActivity">
 
-    <TextView
-        android:layout_width="wrap_content"
+    <com.google.android.material.textfield.TextInputLayout
+        android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:text="Hello World!"
```

Comment:
I WANT THIS TO STAY . DONT DELETE IT IDIOT I AM THE MANAGER STUPID

Reply:
DONT SEND ME THIS BS . YOU ARE FIRED

Reply:
GO WORK IN YOUR MOTHER BASEMENT YOU SLAVE

Reply:
:(